Feat/fake disease

pedro-cze opened the pull request:
Closes #40 

## Description
PR contains the FakeDisease implementation + unit tests

## Technical Details
data class FakeDisease

## Code Samples

```kotlin
fun main() {
        val fakeDisease=  FakeDisease(diseaseName = "COVID-19")
        fakek { println("fakeDisease: ${fakeDisease.diseaseName}") }
}
```

Reviewers: @CodyEngel

pedro-cze committed:
#40 - Add Fake Disease
- FakeDisease, FakeDiseaseTest.kt

pedro-cze committed:
#40 - Add Fake Disease
- typo fix

pedro-cze committed:
Merge branch 'main' into feat/FakeDisease

pedro-cze committed:
#40 - Add Fake Disease
- trailing spaces removed

pedro-cze committed:
#40 - Add Fake Disease
- fixup after merge

CodyEngel commented:
Hey @pedro-cze thanks for creating the PR, [can you use the following PR template](https://github.com/CodyEngel/fakek/blob/main/.github/PULL_REQUEST_TEMPLATE/enhancement_pull_request_template.md)? This helps me out when it comes time to cut a release and eventually when this project has documentation.

pedro-cze commented:
Hi @CodyEngel sorry for missing that template.

CodyEngel committed:
Merge branch 'main' into feat/FakeDisease

Commit:
Remove white space after merge commit

Merge commit introduced trailing whitespace from using the GitHub merge tool, this removes that whitespace and resolves a detekt error.

CodyEngel reviewed (approved):
LGTM thank you @pedro-cze.
